### PR TITLE
clarify that list input->unnamed list output

### PR DESCRIPTION
**Pull request by MichaelChirico**

### CODE REVIEW

File: man/transpose.Rd [hunk 1]
```diff
@@ -26,6 +26,8 @@ transpose(l, fill=NA, ignore.empty=FALSE, keep.names=NULL, make.names=NULL)
 }
 \value{
   A transposed \code{list}, \code{data.frame} or \code{data.table}.
+
+  \code{list} outputs will be unnamed.
```
Comment (ben-schwen) [inline man/transpose.Rd:30]: Not sure we can really say this although I understand the intention.

``` r
transpose(list(names=c('x','y'), c(1,2), c(3,4)), make.names="names")
#> $x
#> [1] 1 3
#> 
#> $y
#> [1] 2 4
```

Comment (MichaelChirico) [reply man/transpose.Rd:30]: how about 

```suggestion
  \code{list} outputs will only be named according to \code{make.names}.
```

Comment (tdhock) [reply man/transpose.Rd:30]: would be great to add that as an example too if there is not one yet.

**Review by ben-schwen — APPROVED** [review]
LGTM

**Pull request merged**